sql/core/src/test/resources/sql-tests/results/json-functions.sql.out

```diff
@@ -26,13 +26,13 @@ Extended Usage:
        {"time":"26/08/2015"}
       > SELECT to_json(array(named_struct('a', 1, 'b', 2));
        [{"a":1,"b":2}]
-      > SELECT to_json(map('a',named_struct('b',1)));
+      > SELECT to_json(map('a', named_struct('b', 1)));
```

Comment:
I think you committed unrelated change?

Reply:
Or you forget to commit `json-functions.sql`?

Reply:
umm. I modified `ExpressionDescription` of `StructsToJson` at @HyukjinKwon 's suggestions which didn't be merged in last PR. Here's the test for `describe function extended to_json`, so I needed to regenerate the golden file for it. So this change isn't from `json-functions.sql`.

Reply:
Oh. I see.